### PR TITLE
Setup bookings controller and booking form

### LINKED ISSUE #27: Create a Booking Form
- Bookings#new method
-  views/bookings/new.html.erb file
- Form to create bookings: (start_date, end_date etc)

### PR DESCRIPTION
**Pull request by johnjoubert**
Closes #27 
Relates to  #25 

I've built out the booking controller **without** #delete, #update, #edit and #show so that @solowresidual can start on the show page.

And I've created a booking form without styling.
Screenshot 2020-11-18 at 11 56 14

I'll open a second PR to finish the controller.

### CODE REVIEW

**Comment by johnjoubert** [comment]
@solowresidual @ceinwenkay @b-lauren please review 🙏

File: db/migrate/20201117232135_change_approved_in_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeApprovedInUsers < ActiveRecord::Migration[6.0]
+  def change
+    change_column_default :bookings, :approved, from: nil, to: false
```
Comment (solowresidual) [inline db/migrate/20201117232135_change_approved_in_users.rb:3]: Is this where you auto set a bookings approved status to false?

**Comment by solowresidual** [comment]
This looks good to me John

File: db/migrate/20201117232135_change_approved_in_users.rb (hunk 1 above)
Comment (johnjoubert) [reply db/migrate/20201117232135_change_approved_in_users.rb:3]: Yeah, that's the default value of false 👍

**Pull request merged**